packages/dashboard-grid-layout/package.json

```diff
@@ -5,14 +5,13 @@
     "@microsoft/load-themed-styles": "^1.7.13",
     "@types/react": "16.3.16",
     "@types/react-dom": "16.0.5",
-    "@types/react-grid-layout": "^0.16.4",
     "@uifabric/example-app-base": ">=6.2.1 <7.0.0",
     "@uifabric/experiments": ">=6.13.1 <7.0.0",
     "css-loader": "^0.15.1",
     "office-ui-fabric-react": ">=6.25.1 <7.0.0",
     "react": "^16.4.1",
     "react-dom": "^16.4.1",
-    "react-grid-layout": "^0.16.6",
+    "react-grid-layout": "samiyaakhtar/react-grid-layout",
```

Comment:
> "samiyaakhtar/react-grid-layout", [](start = 24, length = 34)

Is this right?

Reply:
Yes, that's the forked RGL we're trying to use since drag api is not supported in original RGL. This is for the add card feature. Looking for feedback on the best way to use a modified fork from a library.

Reply:
hmmm. This doesn't seem correct.

Are you planning to submit a PR back to the original package? Is this a temporary fix?